### PR TITLE
feat: upgrade to auth v2.164.0

### PR DESCRIPTION
**Pull request by J0**
Upgrade Auth version to v2.164.0

### CODE REVIEW

**Comment by hf** [comment]
~Seems like something is off here?~

File: common.vars.pkr.hcl [hunk 1]
```diff
@@ -1 +1 @@
-postgres-version = "15.1.1.94"
+postgres-version = "15.1.1.95"
```
Comment (hf) [inline common.vars.pkr.hcl]: Can you check `ansible/vars.yaml` not sure if it's on this branch.

Comment (J0) [reply common.vars.pkr.hcl]: Could you check again? It's showing up on my end

image

**Review by samrose — CHANGES_REQUESTED** [review]
@J0 @hf in this branch if you are targeting 15.6 you will actually need to bump the version in https://github.com/supabase/postgres/blob/release/15.6/common-nix.vars.pkr.hcl

**Review by samrose — APPROVED** [review]

**Pull request merged**